packages/web-service/src/pages/ecologist-experience/enter-license-details/enter-license-details.njk

```diff
@@ -0,0 +1,27 @@
+{% extends "standard-form.njk" %}
+
+{% from "govuk/components/input/macro.njk" import govukInput %}
+
+{% set errorMap = {
+        'enter-license-details': {
+          'string.empty': { ref: '#enter-license-details', text: 'You have not entered a licence number' }
+        }
+    }
+%}
+
+{% set title = "Enter the previous badger mitigation license details" %}
+
+{% block pageContent %}
+
+  {{ govukInput({
+    id: "enter-license-details",
+    name: "enter-license-details",
+    hint: {
+      text: "Enter the reference number of any previously held badger mitigation license."
+    },
+    errorMessage: { text: "Enter the reference number of your previously held licence" } if error['enter-license-details'],
+    classes: "govuk-input--width-10",
+    spellcheck: false
+  }) }}
+
+{% endblock %}
```

Comment:
Watch these end of lines. Not a big deal but the code quality tools like a CR/New line to terminate a file